src/components/Subscriptions/SubscriptionsMonthlyGood.vue

```diff
@@ -248,17 +274,40 @@ export default {
 		result({ data }) {
 			this.isMonthlyGoodSubscriber = data?.my?.autoDeposit?.isSubscriber ?? false;
 			if (this.isMonthlyGoodSubscriber) {
-				const autoDepositAmount = parseFloat(data?.my?.autoDeposit?.amount ?? 0);
-				this.donation = parseFloat(data?.my?.autoDeposit?.donateAmount ?? 0);
+				const activeMonthlyGoodSubscription = data?.mySubscriptions?.values
+					.find(sub => sub?.category?.subscriptionType === 'MG');
+				this.donation = parseFloat(activeMonthlyGoodSubscription?.donation ?? 0);
 				this.dayOfMonth = data?.my?.autoDeposit?.dayOfMonth;
 				this.category = data?.my?.monthlyGoodCategory ?? '';
-				this.mgAmount = autoDepositAmount - this.donation;
+				this.mgAmount = parseFloat(activeMonthlyGoodSubscription?.amount ?? 0);
 				this.paymentMethod = data?.my?.autoDeposit?.paymentMethod ?? {};
 				this.subscriptionId = data?.my?.autoDeposit?.id;
+				this.firstName = data?.my?.userAccount?.firstName ?? '';
+				this.subscriptionsLoans = data?.mySubscriptions?.values?.length ?? 0;
+				// eslint-disable-next-line max-len
+				const mgSubs = activeMonthlyGoodSubscription?.history?.values ?? [];
+				this.subStartTimestamp = mgSubs?.[mgSubs.length - 1]?.timestamp ?? null;
```

Comment:
This is not working for me. I have a new subscription and it displays "12 months". Your conversion is working, but the timestamp on my last entry in `history?.values` is in fact 12 months ago. I would double check what events trigger these timestamps, and if you can rely on the last timestamp to be the subscription creation date. 

I am not confident on what events trigger timestamps. Maybe you want the first timestamp item?